setup.py

```diff
@@ -157,7 +157,7 @@ def build_extension(self, ext: CMakeExtension):
         )
 
         # Ensure that catalyst shared object is copied to the build directory for pip editable install
-        if backend in ("lightning_kokkos"):
+        if platform.system() == "Linux" and backend in ("lightning_kokkos"):
```

Comment:
I think this has already been fixed in another PR?

Reply:
yea, this line as updated.